src/include/stir/DynamicDiscretisedDensity.h

```diff
@@ -162,6 +162,14 @@ class DynamicDiscretisedDensity: public ExamData
 
   const float get_calibration_factor() const;
 
+  //! at method
+  const singleDiscDensT & at(const unsigned int frame_num) const
+  { return this->get_density(frame_num); }
+
+  //! at method
+  const singleDiscDensT & at(const unsigned int frame_num)
```

Comment:
euh. Either we don't have this method, or we return a non-`const` ref. I guess the latter as that's what we have for `get_density`

Reply:
apologies - typo. now fixed